bus/routes.go

```diff
@@ -961,6 +961,15 @@ func (b *Bus) contractIDHandlerPOST(jc jape.Context) {
 	if jc.Check("couldn't store contract", err) == nil {
 		jc.Encode(a)
```

Comment:
Can we flip this into `!= nil` and then return and `jc.Encode(a)` after broadcasting? It looked off to me and then I realised it was because encoding the contract wasn't the final step.